### PR TITLE
feat: send logs to new Cloud Based Sensor bucket

### PR DESCRIPTION
**Pull request by patheard**
# Summary
Update WAF ACL, load balancer and VPC logs to use new Cloud Based
Sensor satellite bucket.

Also sorts the ECS task definition JSON alphanumerically by its keys.
This will prevent Terraform plan flip/flops based on key order in the future.

# Related
* cds-snc/cloud-based-sensor#109

### CODE REVIEW

File: config/terraform/aws/kinesis.tf [hunk 1]
```diff
@@ -64,26 +68,32 @@ EOF
...
-  s3_configuration {
-    role_arn   = aws_iam_role.firehose_waf_logs.arn
-    bucket_arn = aws_s3_bucket.firehose_waf_logs_qrcode.arn
+  extended_s3_configuration {
+    role_arn           = aws_iam_role.firehose_waf_logs.arn
+    prefix             = local.cbs_satellite_bucket_prefix
+    bucket_arn         = local.cbs_satellite_bucket_arn
+    compression_format = "GZIP"
   }
 }
```
Comment (dinophile) [inline config/terraform/aws/kinesis.tf:99]: Nitpicky I know. NL missing. :D

File: config/terraform/aws/task-definitions/covid-portal.json [hunk 2]
```diff
@@ -101,4 +103,4 @@
       }
     ]
   }
-]
+]
```
Comment (dinophile) [inline config/terraform/aws/task-definitions/covid-portal.json:106]: And here. Honestly I don't know why GH can't just do this automatically for us by now. :roll_eyes:

**Review by mohamed-cds — COMMENTED** [review]
LGTM. Will let the portal team approve 🚀 ✅

**Comment by sylviamclaughlin** [comment]
Looks good to me as well!

**Review by sylviamclaughlin — APPROVED** [review]
Looks good to me.

**Pull request merged**